### ISSUE TITLE
Is this support CI version 3 ?

### ISSUE THREAD

**Issue by haidarvm** [dwisetiyadi/CodeIgniter-PHP-QR-Code#11]

**Comment by plitris** [comment]
Yes it does work with CI3. Besides, it's a library written with just PHP functions which doesn't have any dependencies on CodeIgniter. At the worst case you would have to change class names to Capitalised letters or something like that, which is not the case.
